pt-BR/mail/chrome/messenger/preferences/downloadactions.dtd

```diff
@@ -1,32 +1,31 @@
-<!ENTITY  downloadactionsWindow.title         "Download Actions">
+<!ENTITY  downloadactionsWindow.title         "Ações de Download">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY  downloadactionsWindow.title         "Ações para baixar">